crates/core/executor/src/syscalls/precompiles/weierstrass/double.rs

```diff
@@ -57,6 +62,6 @@ impl<E: EllipticCurve> Syscall for WeierstrassDoubleAssignSyscall<E> {
     }
 
     fn num_extra_cycles(&self) -> u32 {
-        0
```

Comment:
I actually think that this should be reverted back to 0.

Based on what we found last night, the read and write of the memory slot is done within the same row, so no need to add an additional cycle.

Note that the opcode will need to be modified to have 0 extra cycles.